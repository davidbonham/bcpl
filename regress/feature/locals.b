GET "libhdr"

LET start() BE $(
    LET a = 4
    LET b = 6
    LET c = a + b
    writehex(c, 8)
$)
