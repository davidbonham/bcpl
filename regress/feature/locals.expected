-- stdout -------------------------------------------
0000000A

-- stderr -------------------------------------------

